### ISSUE TITLE
Still active?

### ISSUE THREAD

**Issue by robinsonkwame** [Open-Argumentation/Canary#1]
Is this project still active?

**Comment by siwells** [comment]
Yes it is. The first prototype was developed as part of an undergraduate project and was of limited success. We're currently redeveloping the entire library into a usable piece of Python software with the next few iterations of prototypes being available over the coming summer.
